vscode/package.json

```diff
@@ -242,6 +255,17 @@
               "markdown": "media/walkthroughs/analysis-arguments.md"
             }
           },
+          {
+            "id": "enable-generativeAI",
+            "title": "Enable GenerativeAI",
+            "description": "Enable Generative AI by starting rpc-server",
+            "completionEvents": [
+              "onCommand:konveyor.toggleGenerativeAI"
+            ],
+            "media": {
+              "markdown": "media/walkthroughs/enable-generativeAI.md"
+            }
```

Comment:
Since the kai/rpc-server is a replacement for the analyzer, I would prefer you put all this discussion into the "Override Analyzer Binary"...maybe muck with the title + certainly change the description.